api/ryml.i

```diff
@@ -200,6 +229,29 @@ def parse(buf, **kwargs):
     return _call_parse(parse_csubstr, buf, **kwargs)
 
 
+def emit(tree, id=None):
+    if id is None:
+        id = tree.root_id()
+
+    return emit_malloc(tree, id)
+
+def compute_emit_length(tree, id=None):
+    if id is None:
+        id = tree.root_id()
+
+    return emit_length(tree, id)
+
+def emit_in_place(tree, buf, id=None):
+    if id is None:
+        id = tree.root_id()
+
+    (failed, expected_size) = emit_to_substr(tree, id, buf)
+    if failed:
+        raise IndexError("Output buffer has {} bytes, but emit required {} bytes".format(
+            len(buf), expected_size))
+
+    return memoryview(buf)[:expected_size]
+
```

Comment:
Nice. Thanks for going the extra mile. I was not asking you for `emit_in_place()`, but you did it. Much appreciated.